dvzo/urls.py

```diff
@@ -20,4 +20,5 @@
     path('admin/', admin.site.urls),
     path('', include('train_management.urls')),
     path('accounts/', include('django.contrib.auth.urls')),
+    path('tinymce/', include('tinymce.urls')),
```

Comment:
Sollte tinymce nicht in verschiedenen Routen einfach eingebettet werden? Warum hat er eine eigene route?

Reply:
Wenn ich das richtig verstanden habe, geht es darum, dass die django-tinymce app hier die settings zur verfügung stellt.
Möglicherweise gibt es einen besseren Weg, dies zu machen.